frontend/src/components/contactsForm.tsx

```diff
@@ -12,11 +17,10 @@ interface Relationship {
 }
 
 interface Props {
```

Comment:
We'd want to use something more descriptive here, `CreateContactFormProps`

Reply:
done!